tests/functional/test_new_resolver_hashes.py

```diff
@@ -96,12 +96,9 @@ def test_new_resolver_hash_intersect_from_constraint(
     )
     requirements_txt = script.scratch_path / "requirements.txt"
     requirements_txt.write_text(
-        """
-        base==0.1.0 --hash=sha256:{sdist_hash} --hash=sha256:{wheel_hash}
-        """.format(
-            sdist_hash=find_links.sdist_hash,
-            wheel_hash=find_links.wheel_hash,
-        ),
+        f"""
+        base==0.1.0 --hash=sha256:{find_links.sdist_hash} --hash=sha256:{find_links.wheel_hash}
+        """,  # noqa: E501
```

Comment:
This one could be converted to multiple strings to avoid the E501?

Reply:
I wasn't sure how to do this in a non-clunky way while preserving the new lines and indents. Instead I created a local variable `sdist_hash` as it's used on multiple lines anyway.

This looks safe as `find_links` is a named tuple, so `sdist_hash` is not a property that will change between attribute lookups.